services/web-app/components/nav-tree/nav-tree.module.scss

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright IBM Corp. 2021, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@use '@carbon/react/scss/spacing' as spacing;
+@use '@carbon/react/scss/theme' as theme;
+@use '@carbon/react/scss/type' as type;
+
+.container {
+  // overide Carbon treeview
+  :global(.cds--tree-node) {
+    padding-left: spacing.$spacing-05;
+    background: transparent;
+  }
+
+  :global(.cds--tree-node__label) {
+    @include type.type-style('heading-01');
+
+    position: relative;
+    cursor: pointer;
+  }
+
+  :global(.cds--tree-parent-node__toggle-icon) {
+    display: none;
+  }
+
+  :global(.cds--tree-parent-node__toggle) {
+    position: absolute;
+    right: 0;
+    width: 100%;
+    height: 100%;
+    margin-right: 0;
+  }
+
+  :global(.cds--tree-parent-node__toggle::after) {
+    position: absolute;
+    top: spacing.$spacing-03;
+    right: spacing.$spacing-05;
+    display: block;
+    width: spacing.$spacing-05;
+    height: spacing.$spacing-05;
+    // chevron icon
+    background-image: url("data:image/svg+xml,%3Csvg focusable='false' preserveAspectRatio='xMidYMid meet' xmlns='http://www.w3.org/2000/svg' fill='currentColor' width='16' height='16' viewBox='0 0 16 16' aria-hidden='true' class='cds--tree-node__icon'%3E%3Cpath d='M8 11L3 6 3.7 5.3 8 9.6 12.3 5.3 13 6z'%3E%3C/path%3E%3C/svg%3E%0A");
```

Comment:
Is this something we could break out into a React component instead?

Reply:
Agree this is a little funky, what we really need is to extend the carbon treeview so we can apply a custom icon for the toggle, but until we do that this was the interim solution I came up with. I can see if I can add the icon as a child of TreeNode if the node has children. 🤔